addons/mail/static/tests/message/message_seen_indicator.test.js

```diff
@@ -232,19 +232,19 @@ test("'channel_seen' notification received is correctly handled", async () => {
     await contains(".o-mail-MessageSeenIndicator i", { count: 0 });
     const channel = pyEnv["discuss.channel"].search_read([["id", "=", channelId]])[0];
     // Simulate received channel seen notification
-    pyEnv["bus.bus"]._sendone(channel, "discuss.channel.member/seen", {
-        id: pyEnv["discuss.channel.member"].search([
-            ["channel_id", "=", channelId],
-            ["partner_id", "=", partnerId],
-        ])[0],
-        channel_id: channelId,
-        last_message_id: 100,
-        partner_id: partnerId,
+    pyEnv["bus.bus"]._sendone(channel, "mail.record/insert", {
+        ChannelMember: {
+            id: pyEnv["discuss.channel.member"].search([
+                ["channel_id", "=", channelId],
+                ["partner_id", "=", partnerId],
+            ])[0],
+            seen_message_id: messageId,
+        },
     });
     await contains(".o-mail-MessageSeenIndicator i", { count: 2 });
 });
 
-test("'channel_fetch' notification then 'channel_seen' received are correctly handled", async () => {
+test("mark channel as fetched then as seen from the bus", async () => {
```

Comment:
Would rename test further so it's more obvious on the intended feature being tested:
`"should display message indicator when message is fetched/seen"`.